tools/snpeff_annotate.cwl

```diff
@@ -46,20 +46,26 @@ arguments:
       set -eo pipefail
 
       tar -xzvf $(inputs.ref_tar_gz.path)
-      && java -Xmx30g -jar /snpEff/snpEff.jar
+
+      java -Xmx${
+        var ram = Math.floor(inputs.ram/1.074 - 1);
+        return ram
+      }g -jar /snpEff/snpEff.jar
```

Comment:
Pretty interesting.

Reply:
Yeah, so 72000M != 72GB. This ensures we don't over-commit heap space dynamically